### PR TITLE
[PRM-1115]: Modified LSP and LPP summary card header and footer

### PR DESCRIPTION
**Pull request by BackRowOla**
Screenshot 2022-03-04 at 17 11 01
Screenshot 2022-03-04 at 17 11 49
Screenshot 2022-03-04 at 17 13 23

### CODE REVIEW

File: app/assets/stylesheets/_summary-card.scss [hunk 1]
```diff
@@ -70,6 +79,21 @@
...
+    padding: 0;
+    display: block;
+    white-space: nowrap;
+
+    @include govuk-media-query($from: desktop) {
+      display: inline-block;
+      margin-left: govuk-spacing(2);
+      text-align: right;
+    }
+  }
```
Comment (LewisMitchell) [inline app/assets/stylesheets/_summary-card.scss:95]: I guess you've rebased from my branch here? So I won't merge until the stylesheet changes are in

File: app/views/components/summaryCardLSP.scala.html [hunk 2]
```diff
@@ -71,14 +67,12 @@ <h3 class="app-summary-card__title">
...
     </div>
     <footer class="app-summary-card__footer">
-        <div class="app-summary-card__actions">
-            <ul class="app-summary-card__actions-list">
+        <div class="app-summary-card__link">
             @{
                 if(!summaryCard.isAppealedPoint) {
                     if(summaryCard.isAddedPoint) {
-                            <li class="app-summary-card__actions-list-item">{messages("summaryCard.adjustedPoint.cannotAppeal")}</li>
+                            <span class="govuk-body">{messages("summaryCard.adjustedPoint.cannotAppeal")}</span>
```
Comment (LewisMitchell) [inline app/views/components/summaryCardLSP.scala.html:74]: Should we use a span here as spans are phrasing content, or would `<div>` or `<p>` make more sense?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/_summary-card.scss (hunk 1 above)
Comment (AndrewRButcher) [reply app/assets/stylesheets/_summary-card.scss:95]: Stylesheet changes have been merged so this PR can be progressed

Comment (BackRowOla) [reply app/assets/stylesheets/_summary-card.scss:95]: 👍

File: app/views/components/summaryCardLSP.scala.html (hunk 2 above)
Comment (LewisMitchell) [reply app/views/components/summaryCardLSP.scala.html:74]: Could you apply something like this?
https://github.com/hmrc/penalties-prototype/blob/f3675c609dc4a4ddccc920c88ab8be967dbb07d3/app/views/beta-baseline/penalties-scenario-adj.html#L97-L99

Comment (BackRowOla) [reply app/views/components/summaryCardLSP.scala.html:74]: Okay that works, doesn't seem to affect the screen reader either 👍

**Comment by LewisMitchell** [comment]
Could you rebase please?

**Comment by LewisMitchell** [comment]
The AT's will need to be updated also

**Comment by AndrewRButcher** [comment]
PR for AT changes https://github.com/hmrc/penalties-acceptance-tests/pull/127

**Comment by LewisMitchell** [comment]
Could you rebase and then I think it's ready

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**